src/invoker.ts

```diff
@@ -64,6 +67,32 @@ function isHttpFunction(
   return functionSignatureType === SignatureType.HTTP;
 }
 
+/**
+ * Checks whether the given user's function is an EVENT function.
+ * @param fn User's function.
+ * @param functionSignatureType Type of user's function signature.
+ * @return True if user's function is an EVENT function, false otherwise.
+ */
+function isEventFunction(
```

Comment:
Can we not have these helper methods?

We should directly check if `functionSignatureType === SignatureType.EVENT`.

We don't use `fn`.